src/resources/Sources/SourcesSubInterfaces/Information.ts

```diff
@@ -0,0 +1,58 @@
+import {ActivityOperation, OperationType, SourceStatusType} from '../../Enums';
+
+export interface SourceCurrentOperation {
+    itemsAdded: number;
+    itemsFiltered: number;
+    itemsRemoved: number;
+    itemsUnchanged: number;
+    itemsUpdated: number;
+    operationFrequencies: [];
```

Comment:
What sorcery is this? is it supposed to be a `string[]` ?

Reply:
Good Question, I check this morning ;)